src/ManageCourses.Api/Controllers/PublishController.cs

```diff
@@ -42,6 +44,25 @@ public async Task<ActionResult> PublishCourseToSearchAndCompare(string providerC
 
             return Ok(enrichmentResult);
         }
+
+        /// <summary>
+        /// Publishes a single course
+        /// </summary>
+        /// <returns>boolean indicating success/failure</returns>
+        [HttpPost]
+        [Route("internal/course/{providerCode}/{courseCode}")]
+        [ProducesResponseType(typeof(bool), 200)]
+        [ProducesResponseType(400)]
+        [ProducesResponseType(404)]
+        [ExemptFromAcceptTerms]
+        [BackendApiTokenAuth]
+        public async Task<ActionResult> PublishCourseToSearchAndCompare(string providerCode, string courseCode, [FromBody]BackendRequest request)
```

Comment:
I think we should give it a different name rather than overloading `PublishCourseToSearchAndCompare()`

How about `JustPushCourseToSearchAndCompare()`

Reply:
The need for verboseness?

Reply:
`Publish Course To Search And Compare` vs
`Just Push Course To Search And Compare` where by the latter is fundamental the same as the former?

Reply:
It's just a suggested name, the key thing is I think the overload provides opportunity for future confusion.

Reply:
I'd like distinct names before I give this the thumbs up

Reply:
You can have `InternalPublishCourseToSearchAndCompare`